staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -766,6 +766,9 @@ func describePod(pod *corev1.Pod, events *corev1.EventList) (string, error) {
 		if len(pod.Spec.PriorityClassName) > 0 {
 			w.Write(LEVEL_0, "Priority Class Name:\t%s\n", stringOrNone(pod.Spec.PriorityClassName))
 		}
+		if pod.Spec.RuntimeClassName != nil && len(*pod.Spec.RuntimeClassName) > 0 {
+			w.Write(LEVEL_0, "Runtime Class Name:\t%s\n", stringOrNone(*pod.Spec.RuntimeClassName))
```

Comment:
we are already checking length of RuntimeClassName and currently `stringOrNone` is redundant in here because it also relies on `len(pod.Spec.RuntimeClassName)`?

Reply:
Thanks for the review!

I've removed `stringOrNone` for both lines `770` and `767`.